src/program/lwaftr/run/run.lua

```diff
@@ -124,9 +124,10 @@ end
 -- Requires a V4V6 splitter iff:
 --   Always when running in on-a-stick mode, except if v4_vlan_tag != v6_vlan_tag.
 local function requires_splitter (opts, conf)
-   if not opts["on-a-stick"] then return false end
-   if not conf.vlan_tagging then return true end
-   return conf.v4_vlan_tag == conf.v6_vlan_tag
+   if not opts["on-a-stick"] then return true end
+   local internal_interface = conf.softwire_config.internal_interface
+   local external_interface = conf.softwire_config.external_interface
```

Comment:
In that other case there are many uses of these locals, so they are useful.

In this case, my second suggestion at least has two uses of `softwire_config`. :-)

True, the resulting line is longer than 80 chars. Do we have guidelines? The codebase has many lines longer than that.